code/modules/mob/inventory.dm

```diff
@@ -158,7 +158,7 @@
 
 	if(isturf(I.loc) && !ignore_anim)
 		I.do_pickup_animation(src)
-	if(get_item_for_held_index(hand_index) != null)
```

Comment:
Can this be 0?

Reply:
yes but only if hand_index is out of bounds for held_items

Reply:
hhhhhh